apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -74,7 +83,41 @@ export default class LevelDetailsDialog extends Component {
         </div>
       );
     } else {
-      return <div>Not implemented yet</div>;
+      // TODO: calculate more of these parameters based on the level and pages
+      return (
+        <UnconnectedTopInstructions
+          hasContainedLevels={false}
+          noVisualization={true}
+          isMinecraft={false}
+          isBlockly={false}
+          isRtl={false}
+          longInstructions={level.longInstructions || level.long_instructions}
+          shortInstructions={level.shortInstructions}
+          noInstructionsWhenCollapsed={true}
+          levelVideos={level.videos}
+          mapReference={level.mapReference}
+          referenceLinks={level.referenceLinks}
+          teacherMarkdown={level.teacherMarkdown}
```

Comment:
Does the teacher markdown show up? The example in the video looked like it was just the instructions but it would be good if the other tabs are working too

Reply:
I used storybook locally to look at it and it does. Awesome! I also see what you mean about the weird styling.

Reply:
Yep, sorry that wasn't in the screenshot. I didn't want the weird styling on storybook to be distracting